stacks/container-apps/augury-alarms.yml

```diff
@@ -0,0 +1,46 @@
+# stacks/container-apps/augury-alarms.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Augury alarms
+Parameters:
+  OpsWarnMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  OpsFatalMessagesSnsTopicArn:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  AuguryWorkerLogGroupName:
+    Type: String
+Resources:
+  AuguryActualsIngestJobInvocationMetric:
```

Comment:
Couldn't find a good example of this in the webapps stack, Though the dovetail stack was somewhat helpful with the metric filters.  I ended up putting the metric right here next to the alarm. Is there a better spot?